pom.xml

```diff
@@ -0,0 +1,394 @@
+                    <configuration>
+                        <junitxml>.</junitxml>
+                        <debugForkedProcess>${maven.scalatest.debug}</debugForkedProcess>
+                        <filereports>TestSuite.txt</filereports>
+                        <testFailureIgnore>false</testFailureIgnore>
+                        <wildcardSuites>${testClasses}</wildcardSuites>
+                    </configuration>
+                    <executions>
+                        <execution>
+                            <id>test</id>
+                            <phase>test</phase>
+                            <goals>
+                                <goal>test</goal>
+                            </goals>
+                            <configuration>
+                                <reportsDirectory>${project.build.directory}/scalatest/reports</reportsDirectory>
+                                <htmlreporters>${project.build.directory}/scalatest/html</htmlreporters>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-jar-plugin</artifactId>
+                    <version>3.0.2</version>
+                    <configuration>
+                        <archive>
+                            <index>true</index>
+                            <manifest>
+                                <addClasspath>true</addClasspath>
+                            </manifest>
+                            <manifestEntries>
+                                <Application-Name>${project.name}</Application-Name>
+                                <Application-Version>${project.version}</Application-Version>
+                                <Implementation-Title>${project.name}</Implementation-Title>
+                                <Implementation-Vendor>Expedia</Implementation-Vendor>
+                                <Build-Jdk>${java.version}</Build-Jdk>
+                                <Build-Time>${maven.build.timestamp}</Build-Time>
+                            </manifestEntries>
+                        </archive>
+                    </configuration>
+                </plugin>
+
+                <plugin>
+                    <groupId>com.github.os72</groupId>
+                    <artifactId>protoc-jar-maven-plugin</artifactId>
+                    <version>${protobuf-maven-plugin.version}</version>
+                    <executions>
+                        <execution>
+                            <phase>generate-sources</phase>
+                            <goals>
+                                <goal>run</goal>
+                            </goals>
+                            <configuration>
+                                <includeDirectories>
+                                    <include>${project.basedir}/../haystack-idl/proto</include>
```

Comment:
`${project.basedir}/../haystack-idl/proto` is outside of this project

Reply:
`${project.parent.basedir}/haystack-idl/proto` works however, if we don't add sub-sub modules